cluster/ubuntu/util.sh

```diff
@@ -14,55 +14,65 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# A library of helper functions that each provider hosting Kubernetes must implement to use cluster/kube-*.sh scripts.
+# A library of helper functions that each provider hosting Kubernetes must
+# implement to use cluster/kube-*.sh scripts.
 set -e
 
-SSH_OPTS="-oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null -oLogLevel=ERROR"
+MASTER=
+MASTER_IP=
+NODE_IPS=
 
-MASTER=""
-MASTER_IP=""
-NODE_IPS=""
+
+BASE_DIR="$KUBE_ROOT/cluster/ubuntu"
+
+SSH_OPTS="\
+ -oStrictHostKeyChecking=no\
+ -oUserKnownHostsFile=/dev/null\
+ -oLogLevel=ERROR\
+"
 
 # Assumed Vars:
 #   KUBE_ROOT
 function test-build-release {
   # Make a release
-  "${KUBE_ROOT}/build/release.sh"
+  "$KUBE_ROOT/build/release.sh"
```

Comment:
I can't find the devel doc that makes it explicit, but we're trying to hew to the Google bash style guide, which is pretty prescriptive here: https://google-styleguide.googlecode.com/svn/trunk/shell.xml . "Prevailing file style" wins in a lot of cases, but please don't un-conform a file, if possible.